src/prefect/cli/deploy.py

```diff
@@ -814,15 +792,41 @@ def _construct_schedule(
         ci: Disable interactive prompts if True
 
     Returns:
-        A schedule object
+        A list of schedule objects
     """
-    schedule = deploy_config.get("schedule", NotSet)
-    cron = get_from_dict(deploy_config, "schedule.cron")
-    interval = get_from_dict(deploy_config, "schedule.interval")
-    anchor_date = get_from_dict(deploy_config, "schedule.anchor_date")
-    rrule = get_from_dict(deploy_config, "schedule.rrule")
-    timezone = get_from_dict(deploy_config, "schedule.timezone")
-    schedule_active = get_from_dict(deploy_config, "schedule.active", True)
+    schedule_configs = deploy_config.get("schedules", NotSet)
+
+    if schedule_configs is not NotSet:
+        schedules = [
+            _schedule_config_to_deployment_schedule(schedule_config)
+            for schedule_config in schedule_configs
+        ]
+    elif schedule_configs is NotSet:
+        if (
+            not ci
+            and is_interactive()
+            and confirm(
+                "Would you like to configure schedules for this deployment?",
+                default=True,
+                console=app.console,
+            )
+        ):
+            schedules = prompt_schedules(app.console)
+        else:
+            schedules = []
```

Comment:
Could we move this prompt inside `prompt_schedules`? I think all the schedule related prompts will be in one function if we do.